app/models/Amounts.scala

```diff
@@ -1,48 +1,54 @@
 package models
 
-import io.circe.{ Decoder, Encoder }
-import io.circe.generic.auto._
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.auto._
+import io.circe.generic.extras.semiauto._
+import io.circe.{Decoder, Encoder}
 
-case class AmountsSelection(
+case class AmountValuesObject(
     amounts: List[Int], 
     defaultAmount: Int,
-    hideChooseYourAmount: Option[Boolean],
+    hideChooseYourAmount: Boolean,
 )
 
-case class ContributionAmounts(
-    ONE_OFF: AmountsSelection,
-    MONTHLY: AmountsSelection,
-    ANNUAL: AmountsSelection,
+case class AmountsCardData(
+    ONE_OFF: AmountValuesObject,
+    MONTHLY: AmountValuesObject,
+    ANNUAL: AmountValuesObject,
 )
 
-case class AmountsTestVariant(
-    name: String,
-    amounts: ContributionAmounts,
+case class AmountsVariant(
+  variantName: String,
+  defaultContributionType: String,
+  displayContributionType: List[String],
+  amountsCardData: AmountsCardData,
 )
 
 case class AmountsTest(
-    name: String,
-    isLive: Boolean,
-    variants: List[AmountsTestVariant],
-    seed: Int,
+  testName: String,
+  liveTestName: Option[String],
+  testLabel: Option[String],
+  isLive: Boolean,
+  region: String,
+  country: List[String],
```

Comment:
here's a proposal for an alternative way to model targeting to make it clear only one is expected:
https://github.com/guardian/support-admin-console/pull/479